src/v/cluster/tx_errc.cc

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "cluster/tx_errc.h"
+
+#include <fmt/format.h>
+#include <fmt/ostream.h>
+
+#include <iostream>
+
+namespace cluster::tx {
+
+std::ostream& operator<<(std::ostream& o, errc err) {
+    switch (err) {
+    case errc::none:
+        return o << "errc::none";
```

Comment:
I'm concerned we removed "tx" from these output strings, is it always going to be clear that it's a `tx::errc`, not any other errc?

Reply:
yes, agree, will change this